### PR TITLE
Added optional delimiter to csv related functions. ";" is the default

### PR DESCRIPTION
**Pull request by hoffigk**

### CODE REVIEW

**Comment by DracoBlue** [comment]
Thanks! Will merge it later!

**Comment by hoffigk** [comment]
Would it be smart to make Craur with this PR php 8 compatible? Or rather with an other PR?

**Comment by DracoBlue** [comment]
Another one please

**Comment by llupa** [comment]
@hoffigk would it be possible to rebase with master and update this PR? I am interested in using other delimiters too and the project now supports PHP 8

File: README.md [hunk 1]
```diff
@@ -408,6 +414,13 @@ element, you can do this:
...
     self;http://example.org/feed.atom
 
+Furthermore, you can also specify the csv delimiter by passing `--csv_output_delimiter` as option 
+
+    $ cat php/tests/fixtures/example_atom_feed.xml | php/craur --output_format csv --csv_output_delimiter , feed.link[].@rel feed.link[].@href
+    // output:
+    alternate,http://example.org/
+    self,http://example.org/feed.atom
+
 ## Changelog
```
Comment (DracoBlue) [inline README.md:425]: @hoffigk  can you add CHANGELOG entry for 3.1.0 ?

**Comment by DracoBlue** [comment]
Thanks a lot!

**Pull request merged**